src/Microsoft.Health.Fhir.Core/Registration/IFhirRuntimeConfiguration.cs

```diff
@@ -7,6 +7,21 @@ namespace Microsoft.Health.Fhir.Core.Registration
 {
     public interface IFhirRuntimeConfiguration
     {
+        string DataStore { get; }
+
+        /// <summary>
+        /// Selective Search Parameter.
+        /// </summary>
         bool IsSelectiveSearchParameterSupported { get; }
+
+        /// <summary>
+        /// Export background worker.
+        /// </summary>
+        bool IsExportBackgroundWorkedSupported { get; }
+
+        /// <summary>
+        /// Customer Key Validation background worker keeps running and checking the health of customer managed key.
+        /// </summary>
+        bool IsCustomerKeyValidationBackgroudWorkerSupported { get; }
```

Comment:
Is this a setting for managed service?

Reply:
Yeap. This is the flag to enable the background services I mentioned to you about. 

This configuration should always be disabled for GEN1, but for GEN2 it'll depend if the FHIR accounts has CMK enabled or not.